modules/flashrom

```diff
@@ -1,34 +1,39 @@
 modules-$(CONFIG_FLASHROM) += flashrom
```

Comment:
We might need to separate flashrom module from dasharo-flashrom. Not all older boards will be able to have this new flashrom version which is bigger.

Reply:
Z690 chipset is available in upstream flashrom already: https://github.com/flashrom/flashrom/blob/master/chipset_enable.c#L2178

Reply:
@miczyg1 : build of master flashrom happening under https://app.circleci.com/pipelines/github/tlaurion/heads/1604/workflows/f7a74cfc-72f2-4de7-aedc-e8a7c24cc5aa for commit https://github.com/tlaurion/heads/commit/0d4997952cbc63a69ed24f3c7daf31b02143c544 (building Heads flashrom to be as flashrom https://github.com/flashrom/flashrom/commit/f3c21c6439f8bbd2911f120c55f301982ecd35bc)

Will edit this commit for size comparison to explain my point once build is finish

Reply:
Clearer and direct test is happening here:
- flashrom/flashrom master on top of heads master: https://app.circleci.com/pipelines/github/tlaurion/heads/1607/workflows/23f6c9f3-926f-478f-a709-70be6d87bae4
- dasharo/flashrom (WP+ast1100 support) on top of heads master: https://app.circleci.com/pipelines/github/tlaurion/heads/1605/workflows/26753865-ac5a-4097-8e06-eccc85d748bf

Will edit this once builds are finished

Reply:
So let's take a clean view now into space differences comparing actual heads master using old flashrom/flashrom and dasharo/flashrom.

Will edit with results.